File: OOP/Aula_07-Herenca/Banco.java

public class Banco {

    private Conta filiados = new Conta[0]; 

    public 

    public void adicionarConta (double saldo_p) {

        Conta nova = new Conta();
        Conta novos = new Conta[this.filiados.lenght + 1];

        for (int i = 0; i < this.filiados.lenght) {
            novos[i] = this.filiados[i];
        }

        novos[this.filiados.lenght] = nova;
        this.filiados = novos;
    }

    public Conta pegarConta (int x) {

        if (x < this.filiados.lenght) {
            return this.filiados[x];
        }
        return null
    }

    public int totalDeContas(){
        return this.filiados.lenght;
    }

}

File: OOP/Aula_09_Interface/Att_02_e_03/Banco.cs
using System;
using Entities;
public class Banco
{
    private Conta[] filiados = Array.Empty<Conta>(); 

    public void AdicionarConta (Conta nova)
    {
        Conta[] novos = new Conta[TotalDeContas() + 1];

        for (int i = 0; i < TotalDeContas(); i++)
            novos[i] = filiados[i];

        novos[TotalDeContas()] = nova;
        filiados = novos;
    }
    public Conta PegarConta (int x)
    {
        if (x < TotalDeContas())
            return filiados[x];

        return null;
    }
    public int TotalDeContas()
    {
        return filiados.Length;
    }
}
